pom.xml

```diff
@@ -0,0 +1,23 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
In general, please apply formatting to the file (add proper indentation, separate the dependencies' block from the common block with groupId, modelVersion, etc.).